src/preferences/dialog/dlgprefinterface.cpp

```diff
@@ -256,8 +256,10 @@ void DlgPrefInterface::slotUpdate() {
     int inhibitsettings = static_cast<int>(m_mixxx->getInhibitScreensaver());
     comboBoxScreensaver->setCurrentIndex(comboBoxScreensaver->findData(inhibitsettings));
 
-    checkBoxAdoptMetaknobPosition->setChecked(m_pConfig->getValue(
-            ConfigKey("[Effects]", "AdoptMetaknobPosition"), true));
+    bool effectLoadMetaknobBehavior = m_pConfig->getValue(
+            ConfigKey("[Effects]", "EffectLoadMetaknobBehavior"), true);
```

Comment:
In general I prefer a bool config value, according to my comment above AdoptMetaknobValue.
If you think we need a third option in future. We need an enum type, for now with two elements.